config/environments/test.rb

```diff
@@ -14,7 +14,11 @@
   config.public_file_server.headers = { 'Cache-Control' => 'public, max-age=3600' }
 
   # Separate cache stores when running in parallel
-  config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
```

Comment:
Was this line previously guarding against parallelisation issues?

I think we need to account for:
- leaking between different parallel tests in CI
- leaking between dev and test environments locally (dev and test should not share the same cache space)

For starters we could at least change `redis://localhost:6379/1` to `redis://localhost:6379/2` here, which I think would solve the second issue.

Reply:
Good pickup! It was, but it looks like we haven't used local parallelisation for a long time:
(TEST_ENV_NUMBER is a variable for the parallel_tests gem which has been removed in 4cbc449a51ce4836152984938e1f161976b6a8fe)
On CI, tests are run in separate environments so the first point is safe.

For the second point, I think that suggestion is a very good idea 👍